server.go

```diff
@@ -34,17 +34,19 @@ func verifyToken(bot *Bot, w http.ResponseWriter, r *http.Request) {
 }
 
 type receive struct {
+	Object  string  `json:"object"`
```

Comment:
Object is a string and always "page". Not sure if we need this:

https://developers.facebook.com/docs/messenger-platform/webhook-reference#common_format